infra/fastly/terraform/dl-sandbox.k8s.dev/vcl/binaries.vcl

```diff
@@ -18,6 +18,13 @@ sub vcl_recv {
   if (req.method != "HEAD" && req.method != "GET" && req.method != "FASTLYPURGE") {
     return(pass);
   }
+
+  # Drop all the headers except the list providers
+  header.filter_except(req, "Accept", "User-Agent", "Accept-Encoding");
```

Comment:
I don't think we actually want to send these to the backend, I think we want to drop all of them?

It would be nice if the CDN side respected gzip request but it shouldn't care about User-Agent, and we shouldn't need to pass any of it to GCS.